src/node.cc

```diff
@@ -2935,6 +2942,12 @@ static void PrintHelp() {
          "  -p, --print          evaluate script and print result\n"
          "  -i, --interactive    always enter the REPL even if stdin\n"
          "                       does not appear to be a terminal\n"
+#ifdef _WIN32
+         "  -r, --require        ';'-separated list of modules to"
```

Comment:
Using a colon for this doesn't work on windows. 
+1 to what the PR makes of it.